server/controllers/article.js

```diff
@@ -166,6 +166,35 @@ class Article {
     }
     return goodHttpResponse(response, 401, 'You cannot deleted this article');
   }
+
+  /**
+   * Update an article
+   * @param {object} request Request Object
+   * @param {object} response Response Object
+   * @returns {object} User Object
+   */
+  static async updateArticle(request, response) {
+    const { id } = request.article;
+    const {
+      title,
+      body,
+      description,
+      images,
+    } = request.body;
+    const readtime = Math.floor(body.split(' ').length / 3);
+
```

Comment:
@TheoOkafor are we updating every single field when the update function is called even when the only thing that changed is maybe the images or the title?

Reply:
Yes.

Reply:
This is fine

Reply:
Technically,  what we're trying to do here is a `patch`. Please read on the difference between a `put` and a `patch`. Also you should be updating the updatedAt column as well

Reply:
@seyi-adeleke the `updatedAt` column  is updated automatically by sequelize once an update is performed.

Reply:
I have made the required changes.